addon/components/base-focusable.js

```diff
@@ -10,6 +10,7 @@ export default Component.extend(EventsMixin, HasBlockMixin, {
   active: false,
   focused: false,
   hover: false,
+  focusable: true,
```

Comment:
I haven't looked but I thought both checkboxes and switches could be focused and toggled with the keyboard. If focusOnlyOnKey needs a new name to cover all its uses, I think that would be a bit better